wiki/Guides/Beatmapping/Adding_Custom_Hit_Sounds/de.md

```diff
@@ -34,7 +34,7 @@ Abhängig von welcher Art von den Hitsound man haben möchte, sollte der Dateina
 
```

Comment:
change the format to match the English version
aka change bold to code and include file extension into it